Formula/yle-dl.rb

```diff
@@ -99,3 +107,210 @@ def install
     assert_match "2012-05-30T10:51", output
   end
 end
+
+__END__
```

Comment:
This patch is pretty big. We should probably host it at formula-patches.

Reply:
https://github.com/Homebrew/formula-patches/pull/859